odoo_backup_sh_dropbox/README.rst

```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+========================
+ Odoo-backup.sh Dropbox
+========================
+
+The application allows managing database backups via Dropbox, without downloading and uploading an archive via a browser.

```

Comment:
No, it does not. It allows making backups only. Not restoring it via /web/database/backups page
![wrong](https://user-images.githubusercontent.com/186131/60324651-c88bd300-999e-11e9-8715-938d7f14b704.png)